ERCS/erc-7761.md

```diff
@@ -0,0 +1,121 @@
+---
+eip: 7761
+title: Cache invalidation in ERC-5219 mode Web3 URL
+description: In ERC-5219 resolve mode, add mechanisms to alleviate limitations to the use of RFC 9111 HTTP caching
+author: Nicolas Deschildre (@nand2)
+discussions-to: https://ethereum-magicians.org/t/eip-4804-web3-url-to-evm-call-message-translation/8300
```

Comment:
Please create a new topic to discuss this ERC.

Reply:
Thanks for the review, done!